### ISSUE TITLE
New CS proposal: [Network segmentation Cheat Sheet] (draft to release)

### ISSUE THREAD

**Issue by sergiomarotco** [OWASP/CheatSheetSeries#1036]
## What is the proposed Cheat Sheet about?
It would be useful to transfer the draft application segmentation standard ([Network_Segmentation_Cheat_Sheet](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets_draft/Network_Segmentation_Cheat_Sheet.md)) to the release version.

## What security issues are commonly encountered related to this area?
- Segmentation of service components
- Basic allowed network access
- In which segments which elements of the service should be placed

## What is the objective of the Cheat Sheet?
Show best practices for network segmentation on graphical diagrams

#### Are there any other OWASP projects that cover this area?
Nope
#### Would this content be better suited to the testing guide projects?
Nope, not for testing, maybe for compliance auditing
#### What other third party guidelines or resources currently exist?
My :)
- [Network segmentation cheat sheet](https://github.com/sergiomarotco/Network-segmentation-cheat-sheet)
- [ADO Server segmentation cheat sheet](https://github.com/sergiomarotco/Azure-DevOps-Server-segmentation-cheat-sheet)

In summary, I propose to approve #1031

**Comment by sergiomarotco** [comment]
![ED84E75F-7C2E-4879-A460-63B2351DA88C](https://user-images.githubusercontent.com/29877074/210402257-607bf156-4dfd-4459-a740-c6324919b940.jpeg)
@szh How can I fix the display of images on the site?

**Comment by szh** [comment]
Here are the errors in the Chrome dev console:
<img width="1774" alt="image" src="https://user-images.githubusercontent.com/546965/210402710-6173e7a6-c752-452c-a8bc-ebbeae0845e1.png">

**Comment by jmanico** [comment]
My mistake for merging this early. I'll aim to fix this today!

**Comment by jmanico** [comment]
Fixed 61f39b175c32b1bd16278a5512dd9c6aa8fad891
